### ISSUE TITLE
loading-states extension now fails in Safari

### ISSUE THREAD

**Issue by RussBaz** [bigskysoftware/htmx#2450]
Hi. `loading-states` extension stopped working in Safari (I have just updated my macOS to 14.4.1 [Intel] but I can check on M1 as well) but it works correctly in Firefox.

The console indicates that `evt.target` is `null` when the following point in code is reached:
```js
htmx.defineExtension('loading-states', {
        onEvent: function (name, evt) {
            if (name === 'htmx:beforeRequest') {
                const container = loadingStateContainer(evt.target);
```

Which results in `[Error] TypeError: null is not an object (evaluating 'e.closest') — htmx.min.js:1:6518` error displayed in the console.

The htmx version I use is 1.9.11.

Can anyone check why this might be happening or give tips on how to troubleshoot it properly?

Thank you.

UPD: It works as expected in safari (17.3.1) for macOS 14.3.1 (M1)

**Comment by jozefrebjak** [comment]
The same problem on M1 Pro.  

HTMX **1.9.5** and after problem with loading-states I updated to **1.19.11** and problem is still there. 

Current Safari **17.4.1**.

MacOS Sonoma **14.4.1**

Console Log:
![CleanShot 2024-04-12 at 11 30 50](https://github.com/bigskysoftware/htmx/assets/36922215/ba2b97a0-31c6-4c4f-b451-8fa54699397b)

**Comment by andryyy** [comment]
Probably needs the same fix as https://github.com/bigskysoftware/htmx/pull/2391/commits/695fd8aa14404afc695ae96949ce61965fc2b3ae

**Comment by jozefrebjak** [comment]
@andryyy Yes you are right. I created a PR https://github.com/bigskysoftware/htmx/pull/2478

**Comment by 1cg** [comment]
extension issue, no longer relevant in 4.x
